lib/ddtrace.rb

```diff
@@ -80,6 +80,3 @@ module Datadog
 require 'ddtrace/contrib/sinatra/integration'
 require 'ddtrace/contrib/sneakers/integration'
 require 'ddtrace/contrib/sucker_punch/integration'
-
-# Load extensions from CI namespace
-require 'datadog/ci'
```

Comment:
why are we removing the ci namespace extensions from ddtrace.rb?

Reply:
This was autoloading CI; new behavior is to add `require 'datadog/ci'` to your `spec_helper.rb`. This is to prevent CI extensions from interfering with production environments and make for cleaner separation.

Reply:
fair enough, makes sense to me